environment.yml

```diff
@@ -3,23 +3,24 @@ channels:
 dependencies:
   - cvxopt=1.3.2
   - deepdish=0.3.7
-  - discretize=0.10.0
-  - empymod=2.0.6
+  - discretize=0.11.1
+  - empymod=2.4.0
   - git=2.41.0
   - ipywidgets=8.1.0
   - jupyter=1.0.0
-  - matplotlib=3.4.3
-  - mkl=2022.2.1
-  - numpy=1.24.4
-  - pillow=10.0.0
-  - pip=23.2
-  - pymatsolver=0.2.0
-  - python=3.10
+  - matplotlib=3.10.0
```

Comment:
I am not too sure why this doesn't resolve, but it seems that matplotlib is the issue. If I remove this line, it installs correctly on my machine, and through the dependency tree, correctly installs matplotlib. So we could likely remove this explicit dependency and have it just install implicitly??